src/wl_window.c

```diff
@@ -2667,8 +2667,34 @@ void _glfwGetCursorPosWayland(_GLFWwindow* window, double* xpos, double* ypos)
 
 void _glfwSetCursorPosWayland(_GLFWwindow* window, double x, double y)
 {
-    _glfwInputError(GLFW_FEATURE_UNAVAILABLE,
-                    "Wayland: The platform does not support setting the cursor position");
+    if (!_glfw.wl.pointerConstraints)
```

Comment:
Side note: The `lockPointer` function makes also makes use of `pointer-constraints-unstable-v1` functions [here](https://github.com/Friz64/glfw/blob/2307384a83ab87ba454458db0579f0ca318151f8/src/wl_window.c#L2968), but only checks for `relative-pointer-unstable-v1`. Is that a bug?